internal/replay/replay_http_server.go

```diff
@@ -131,3 +147,109 @@ func (r *ReplayHTTPServer) writedResponse(w http.ResponseWriter, resp *store.Rec
 	_, err := w.Write(resp.Body)
 	return err
 }
+
+func (r *ReplayHTTPServer) proxyWebsocket(w http.ResponseWriter, req *http.Request, chunks []string) {
+	clientConn, err := r.upgradeConnectionToWebsocket(w, req)
+	if err != nil {
+		http.Error(w, fmt.Sprintf("Error proxying websocket: %v", err), http.StatusInternalServerError)
+		return
+	}
+	defer clientConn.Close()
+	replayWebsocket(clientConn, chunks)
+}
+
+func (r *ReplayHTTPServer) loadWebsocketChunks(sha string) ([]string, error) {
+	responseFile := filepath.Join(r.recordingDir, sha+".websocket")
+	fmt.Printf("loading websocket response from : %s\n", responseFile)
+	bytes, err := os.ReadFile(responseFile)
+	var chunks = make([]string, 0)
+	if err != nil {
+		fmt.Printf("Error loading websocket response: %v\n", err)
+		return chunks, err
+	}
+
+	i := 0
+	response := string(bytes)
+	for i < len(response) {
+		// Extracts prefix
+		prefix := response[i]
+		if prefix != '>' && prefix != '<' {
+			return nil, fmt.Errorf("invalid message prefix at position %d: expected '>' or '<', got '%c'", i, prefix)
+		}
+		i++ // Move cursor past prefix.
+
+		// Extracts chunk length
```

Comment:
readability nit: consider extracting to a separate function (maybe something like `num, err := consumeNumber(prefix[i], &i)` ?)

Reply:
Done, created a helper method - extractNumber